packages/cloud_firestore/cloud_firestore_web/pubspec.yaml

```diff
@@ -4,27 +4,29 @@ homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/c
 version: 0.3.0+1
 
 environment:
-  sdk: ">=2.12.0-0 <3.0.0"
-  flutter: ">=1.12.13+hotfix.4 <2.0.0"
+  sdk: ">=2.12.0-133.7.beta <3.0.0"
+  flutter: ">=1.9.1+hotfix.5 <2.0.0"
 
 dependencies:
+  cloud_firestore_platform_interface: ^3.0.1
+  firebase_core: ">=0.8.0-1.0.nullsafety.0 <0.8.0-2.0.nullsafety.0"
+  firebase_core_web: ">=0.3.0-1.0.nullsafety.0 <0.3.0-2.0.nullsafety.0"
   flutter:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  http_parser: ^4.0.0-nullsafety
-  meta: ^1.1.7
-  firebase_core: ">=0.8.0-1.0.nullsafety.0 <0.8.0-2.0.nullsafety.0"
-  firebase_core_web: ">=0.3.0-1.0.nullsafety.0 <0.3.0-2.0.nullsafety.0"
-  cloud_firestore_platform_interface: ^3.0.1
   js: ^0.6.1
+  meta: ^1.1.7
+  
+dependency_overrides:
+  http_parser: ^4.0.0-nullsafety  
 
 dev_dependencies:
-  pedantic: ^1.8.0
+  firebase_core_platform_interface: ">=4.0.0-1.0.nullsafety.0 <4.0.0-2.0.nullsafety.0"
   flutter_test:
     sdk: flutter
-  firebase_core_platform_interface: ">=4.0.0-1.0.nullsafety.0 <4.0.0-2.0.nullsafety.0"
   mockito: ^5.0.0-nullsafety.2
+  pedantic: ^1.8.0  
```

Comment:
Pedantic can be removed, no longer in use as we do our own analysis_options.yaml file now.